pt-BR/mail/chrome/mozldap/ldap.properties

```diff
@@ -176,43 +176,43 @@ authPromptText=Forneça a senha para %1$S.
 
 ## @name UNWILLING_TO_PERFORM
 ## @loc none
-53=LDAP server is unwilling to perform
+53=O servidor LDAP não consegue executar
 
 ## @name LOOP_DETECT
 ## @loc none
 54=Loop detectado
 
 ## @name SORT_CONTROL_MISSING
 ## @loc none
-60=Sort Control is missing
+60=O controle de ordenamento está ausente
 
 ## @name INDEX_RANGE_ERROR
 ## @loc none
-61=Search results exceed the range specified by the offsets
+61=Os resultados da pesquisa excedem o intervalo especificado pelos offsets
```

Comment:
CORRECTIONS NEEDED it should be 61=Os resultados da pesquisa excedem o intervalo especificado pelos intervalos